app/forms/phone_helpers.rb

```diff
@@ -0,0 +1,22 @@
+module PhoneHelpers
+  def require_phone_confirmation?
+    phone_changed?
+  end
+
+  def phone_changed?
+    @phone_changed == true
+  end
+
+  private
+
+  def check_phone_change(params)
+    formatted_phone = params[:phone].phony_formatted(
+      format: :international, normalize: :US, spaces: ' '
+    )
+
+    return unless formatted_phone != @user.phone
+
+    @phone_changed = true
+    self.phone = formatted_phone
+  end
+end
```

Comment:
Yes, this could be removed, it's left over from the preference attribute work, but I felt it added more clarity. Removing.